### PR TITLE
Creating an app_test.py test class 

### PR DESCRIPTION
**Pull request by JayCax**
Noticed that app_test.py wasn't structured in a test class. Restructured app_test.py into a test class.

### CODE REVIEW

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
This PR seems to do something else. Can you explain the changes and why they are needed?

CC @gbaned : let's close this PR in ~20 days if no replies arrive.

**Comment by mihaimaruseac** [comment]
Also, please squash the commits, it's not looking good to have so many commits for just a single file change. Commit padding is not looking great :)

**Comment by JayCax** [comment]
Hey all, I wanted to contribute but there is obviously unexpected results. I will be closing the pull request - thank you for your time .

File: tensorflow/python/platform/app_test.py [hunk 1]
```diff
@@ -18,24 +18,48 @@
...
+      app.run()
+    self.assertEqual(cm.exception.code, 2)
+
+  def testWith1AcceptableArg(self):
+    sys.argv.extend(["--myflag"])
+    with self.assertRaises(SystemExit) as cm:
+      app.run()
+    self.assertEqual(cm.exception.code, 2)
+
+  def testWithoutPassThrough(self):
```
Comment (vufg) [inline tensorflow/python/platform/app_test.py:46]: what's the difference between this one and "testWith3AcceptableArgs"?

**Pull request closed without merging**